client/src/slices/draftsSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { fetchUser } from '../actions';
+import { Review } from '../types';
+
+export type DraftState = Review[];
+
+const initialState: Review[] = [];
+
+const draftsSlice = createSlice({
+  name: 'drafts',
+  initialState,
+  reducers: {
+    updateDrafts: (_state, { payload }) => payload,
+  },
+  extraReducers: (builder) => {
```

Comment:
Poorly named IMO, but `extraReducers` can listen to actions that are generated by
- other slices
- the `actions/index.ts` file

In this case, `fetchUser` doesn't "belong" to any single slice, so all slices need to list it as an `extraReducer`.

This uses slightly different syntax, but is otherwise identical to how `reducers` works. Reducer functions declared under `extraReducer` can't be exported from this slice as action creators, since they belong to another file/slice.